common/ocp-config/ds-component-environment/ds-component-oauthproxy.yml

```diff
@@ -149,7 +149,7 @@ objects:
                 - --tls-cert=/etc/tls/private/tls.crt
                 - --tls-key=/etc/tls/private/tls.key
                 - --cookie-secret=${SESSION_SECRET}
-                - --request-logging=true
+
                 - >-
                   --openshift-sar={"namespace":"${PROJECT}-${ENV}","resource":"services","name":"${COMPONENT}-auth-proxy","verb":"get"}
```

Comment:
you only need human login flow handling here, right? no programmatic API access, right?

Reply:
Right, only human login.

Reply:
and actually, I can't leave it in the "request-logging=true", as otherwise I run into websocket issues